spec/lib/hubspot/ticket_properties_spec.rb

```diff
@@ -0,0 +1,41 @@
+describe Hubspot::TicketProperties do
+  describe '.create' do
+    context 'with no valid parameters' do
+      it 'should return nil' do
+        VCR.use_cassette 'ticket_fail_to_create_property' do
+          expect(Hubspot::TicketProperties.create!({})).to be(nil)
+        end
+      end
+    end
+
+    context 'with all valid parameters' do
+      let(:params) do
+        {
+          'name' => 'my_new_property',
+          'label' => 'This is my new property',
+          'description' => 'How much money do you have?',
+          'groupName' => 'ticketinformation',
+          'type' => 'string',
+          'fieldType' => 'text',
+          'hidden' => false,
+          'deleted' => false,
+          'displayOrder' => 0,
+          'formField' => true,
+          'readOnlyValue' => false,
+          'readOnlyDefinition' => false,
+          'mutableDefinitionNotDeletable' => false,
+          'calculated' => false,
+          'externalOptions' => false,
+          'displayMode' => 'current_value'
```

Comment:
ok my previous idea about being nice to users may be a bad one, given the number of parameters